doc-content/enterprise-only/openshift/externaldb-build-proc.adoc

```diff
@@ -6,18 +6,23 @@ If you want to use an external database server for a {KIE_SERVER} and the databa
 Complete the steps in this build procedure to provide drivers for any of the following database servers:
 
 * Microsoft SQL Server
-* MariaDB
 * IBM DB2
 * Oracle Database
 * Sybase
 
+You can also optionally use this procedure to build a new version of drivers for any of the following database servers:
```

Comment:
Minor suggestion - remove "also" and "any of", and maybe move "optionally".

Optionally, you can use this procedure to build a new version of drivers for the following database servers: